packages/react-ui-components/src/Tree/node.css

```diff
@@ -119,6 +136,7 @@
 
     [data-is-drag-happening] & {
         visibility: visible;
+        height: 5px;
```

Comment:
```suggestion
```

This doesn't make a difference, because the drop targets receive paddings, which control the height. `.dropTarget--before` for example has the calculated CSS properties:
```css
box-sizing: border-box;
padding-top: 4px;
padding-bottom: 4px;
```
Which makes it at least `8px` in height. Any explicit `height` below `8px` will be swallowed by `box-sizing: border-box;`.

Instead, changing `padding-top` and `padding-bottom` will increase the height.

Also, moving those declarations below the `[data-is-drag-happening] & { ... }` selector isn't necessary: `visibility: hidden;` of the base declaration in `.dropTarget` makes sure that the drop target containers do not interfere with the pointer events for the tree node. So, `.dropTarget` is allowed to have increased height to begin with.